src/gateway/kong-enterprise/secrets-management/index.md

```diff
@@ -27,6 +27,12 @@ you can reference each secret with a `vault` reference. For example:
 
 In this way, secrets management becomes centralized.
 
+This feature is enabled by default.
+Due to conflicts with previous releases of {{site.base_gateway}},
+the endpoints for secrets management in the
+Admin API have changed from the previous `/vaults-beta` prefix to
+`/vaults` with `vaults_use_new_style_api=on` set in `kong.conf`.
```

Comment:
> Admin API have changed from the previous `/vaults-beta` prefix to
`/vaults` with `vaults_use_new_style_api=on` set in `kong.conf`.

@lena-larionova This option (`vaults_use_new_style_api`) is only added in 2.8.x, and not available in 3.0.x. Gateway 3.0 keeps only the new style vault APIs.
Could you help remove this sentence from 3.0 gateway docs? thanks!

Reply:
Ack - thanks for catching this. I'll make the update.